cypress/support/constants/constants.js

```diff
@@ -137,10 +139,12 @@ module.exports = {
   FCS_VALIDATION_JSON: 'fCSValidationjson',
   FIREBOLT: 'firebolt',
   FIREBOLTCALL: 'fireboltCall',
-  FIREBOLTCALLS_FROM_CONFIGMODULE: './cypress/fixtures/external/',
-  FIREBOLTCALLS_FROM_FCS: './cypress/fixtures/',
+  FIREBOLTCALLS_FROM_CONFIGMODULE: './node_modules/configModule/testData/fireboltCalls/',
+  FIREBOLTCALLS_FROM_FCS: './cypress/fixtures/fireboltCalls/',
   FIREBOLTCONFIG: 'fireboltConfig',
```

Comment:
The path would still be external right ? and not node_modules .

Reply:
After consulting Kurt about retrieving the JSON files from external sources or nodeModules, he confirmed that it's okay to read them from nodeModules. do we need to read it from external instead of nodemodules?